google/resumable_media/_helpers.py

```diff
@@ -45,6 +46,13 @@ def do_nothing():
     """Simple default callback."""
 
 
+def _base_headers(headers):
+    version = pkg_resources.get_distribution("google-resumable-media").version
+    headers["User-Agent"] = "gcloud-python/{}-resumable-media".format(version)
```

Comment:
This should not blindly override headers. At most it should append this to the end of an existing header.

I recommend using `setdefault()` or similar logic next time.